packages/gatsby-theme-api-docs/src/components/api/resource/method/method.js

```diff
@@ -65,95 +71,91 @@ const Method = ({ apiKey, url, resourceUriParameters, method, methodType }) => {
-                css={computeMethodNameBackgroundColor(methodType.toLowerCase())}
-              />
-              <UrlScopesResponses
-                data={{
-                  url,
-                  scopes: {
-                    title: oauth2Scopes,
-                    scopes: method.securedBy
-                      ? method.securedBy[0].oauth_2_0.scopes
-                      : null,
-                  },
-                  responses: method.responses,
-                }}
-              />
-            </SpacingsInline>
-          </UrlScopesResponseOverallContainer>
-
-          {allUriParameters.length > 0 ? (
-            <Parameters
-              title={pathParametersTitle}
-              parameters={allUriParameters}
-            />
-          ) : null}
-
-          {method.queryParameters ? (
-            <Parameters
-              title={queryParametersTitle}
-              parameters={method.queryParameters}
-            />
-          ) : null}
-
-          {method.body ? (
-            <RequestRepresentation
-              titleSuffix={requestRepresentation}
-              apiKey={apiKey}
-              apiType={method.body.applicationjson.type}
-            />
-          ) : null}
-        </SpacingsStack>
+    <SpacingsStack scale="s">
+      <SpacingsInline alignItems="center" scale="s">
+        <Type
+          css={css`
+            background-color: ${methodColor};
+          `}
+        >
+          {methodType}
+        </Type>
+
+        <TitleWithAnchor id={id}>{method.displayName}</TitleWithAnchor>
+      </SpacingsInline>
+
+      <Description>{method.description}</Description>
+
+      <SpacingsStack scale="m">
+        <UrlScopesResponseContainer
+          css={css`
+            border-left-color: ${methodColor};
+          `}
+        >
+          <UrlScopesResponses
+            data={{
+              apiKey,
+              uris,
+              scopes: {
+                title: oauth2Scopes,
+                scopes: method.securedBy
+                  ? method.securedBy[0].oauth_2_0.scopes
+                  : null,
+              },
+              responses: method.responses,
+            }}
+          />
+        </UrlScopesResponseContainer>
+
+        {allUriParameters.length > 0 ? (
+          <Parameters
+            title={pathParametersTitle}
+            parameters={allUriParameters}
+          />
+        ) : null}
+
+        {method.queryParameters ? (
+          <Parameters
+            title={queryParametersTitle}
+            parameters={method.queryParameters}
+          />
+        ) : null}
+
+        {method.body ? (
+          <RequestRepresentation
+            titleSuffix={requestRepresentation}
+            apiKey={apiKey}
+            apiType={method.body.applicationjson.type}
+          />
+        ) : null}
       </SpacingsStack>
-    </div>
+    </SpacingsStack>
   );
 };
 
-function computeMethodNameBackgroundColor(methodName) {
+function computeMethodColor(methodName) {
```

Comment:
Nitpick: rename to `getColorByMethod`

Reply:
Could you please leave out nitpicks when you review?

I don't think it's good investing time in a discussion about something that's subjective.

Reply:
Sorry but nitpicks are still part of a review, and they are not necessarily subjective.

In this case I'm providing a suggestion to have the name following conventions in other repos.

> I don't think it's good investing time in a discussion

Yes it's good to invest time to make the code more readable, and no it's not meant to be a discussion because it's a nitpick and it's up to you to take the suggestion or not.